task5/modules/asg-tfm/main.tf

```diff
@@ -0,0 +1,27 @@
+resource "aws_launch_configuration" "this_launch_conf" {

```

Comment:
Please add tags to all taggable resources

Reply:
c0939e9d5d16b4dd306cbde842b711481bb0dcd6 added